app/models/locations/country.rb

```diff
@@ -24,7 +24,6 @@ class Country < ApplicationRecord
   validates :alpha_3_code, presence: true, uniqueness: true
   validates :numeric_code, presence: true, uniqueness: true
   validates :mernis_code, numericality: { only_integer: true }, uniqueness: true, allow_blank: true
-  validates :yoksis_code, numericality: { only_integer: true }, uniqueness: true, allow_blank: true
```

Comment:
Validasyonu neden sildin?

Reply:
Alan integer olduğu için `numericality: { only_integer: true }` buna gerek olmadığı düşündüm. 

https://github.com/omu/nokul/issues/99#issuecomment-404238753 bu commentteki sorumun cevabı evetse bu alan uniqueness olmuyor.

Reply:
:+1: